src/other/broadcasting.jl

```diff
@@ -0,0 +1,63 @@
+struct LazyNewColDataFrame
+    df::DataFrame
+    col::Symbol
+end
+
+# we allow LazyNewColDataFrame only for data frames with some columns
+Base.axes(x::LazyNewColDataFrame) = axes(x.df[1])
+
+Base.Broadcast.broadcastable(df::AbstractDataFrame) = Matrix(df)
+
+Base.maybeview(df::AbstractDataFrame, idxs...) = view(df, idxs...)
+
+function Base.maybeview(df::AbstractDataFrame, idxs)
```

Comment:
Or just `idx` (singular)?

Reply:
`idxs` may be a collection of column indices so I thought about plural. But I am OK to switch to singular. If felt consistent with the method argument names above in line 11.